### DISCUSSION TITLE
Brackets in semantic tokens that are comments

### DISCUSSION THREAD

**Discussion by phlegmaticprogrammer** [microsoft/vscode-discussions#382] [Extension Development QnA] [answered]
I have implemented a tokeniser for my language solely based on `DocumentSemanticTokensProvider` without any supporting TextMate grammar.

The tokens that are declared as `comment` still show colorised brackets. I've seen that this [has been an issue before](https://github.com/microsoft/vscode/issues/132249) that seems to have been fixed, but it seems for semantic tokens this issue still exists. Does this warrant a bug report, or is this just how it is supposed to be?

Best,

Steven Obua

**Comment by isidorn** [comment, marked as answer]
Filling a new issue in https://github.com/microsoft/vscode/issues is the right way to go here
fyi @hediet

**Comment by hediet** [comment]
There is already an issue out there that requests this feature. Unfortunately, it is not planned for now.

**Reply by phlegmaticprogrammer** [reply]
Ok, thank you for the update. No worries, I decided to switch bracket matching etc. off for my extension. Given that brackets don't really add semantic information, but just clarify it, I actually want brackets to be de-emphasised.
